telos-frontend/src/components/todo/DashboardTodo.jsx

```diff
@@ -1,3 +1,376 @@
-const DashboardToDo = () => <div />;
+/* eslint-disable react/no-this-in-sfc */
```

Comment:
It's generally bad practice to disable eslint rules - what's your reasoning for doing this here?